internal/compiler-bridge/src/main/scala_2.13/xsbt/ConsoleInterface.scala

```diff
@@ -47,7 +47,6 @@ class ConsoleInterface {
               if (loader eq null) super.parentClassLoader
               else loader
           }
-          intp.setContextClassLoader()
```

Comment:
Would this mean using some range of Scala 2.13.x would require some range of future version of sbt?